Library/Homebrew/extend/os/linux/diagnostic.rb

```diff
@@ -0,0 +1,60 @@
+require "tempfile"
+
+module Homebrew
+  module Diagnostic
+    class Checks
+      alias mac_message check_tmpdir_sticky_bit
```

Comment:
I'd suggest the name `generic_check_tmpdir_sticky_bit` rather than `mac_message`.

Reply:
ok